packages/playground/README.md

```diff
@@ -1,3 +1,37 @@
-# playground
+# Playground
 
-This package is not published, and is just used to verify a standalone app
\ No newline at end of file
+Playground is a sample standalone application that shows usage of React Native for Windows vNext with several test scenarios. 
+
+## Launching the app
+1. Make sure your development machine has been set up with all the system requirements and dependencies mentioned [here](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md). Make sure a browser is launched and running.
```

Comment:
change browser to Chrome. currently we only support Chrome

Reply:
That's not true - we are able to run against Edge/Chromium and it works as well.